partials/footer.hbs

```diff
@@ -1,3 +1,3 @@
-<footer>
+<footer class="container footer  col-xs-10 col-xs-offset-1">
```

Comment:
I don't remember sites with different footers between pages. I think the general layout must be consistent. Aren't you?